.changeset/twenty-cobras-push.md

```diff
@@ -0,0 +1,5 @@
+---
+'astro': major
+---
+
+Adds base url to next and prev url in `paginate()` utility
```

Comment:
``````suggestion
Changes the data returned for `page.url.next` and `page.url.prev` to include the value set for `base` in your Astro config. 

Previously, you had to manually prepend your configured value for `base`  to the URL path.  Now, Astro automatically includes your `base` value to `next` and `prev` URLs.

If you are using the `paginate()` function for "previous" and "next" URLs, remove any existing `base` value as it is now added for you:

```diff

show a line of code plz!
```

``````
Assuming this is the change for 5.0, would love to see an example and explicit instruction here showing people a line of code they'll have to change!

Something kind of like this? (Pls make it accurate)

Reply:
I'll commit the suggestion and follow up with a commit that shows the diff